tools/linter_lib/custom_check.py

```diff
@@ -534,24 +534,17 @@ def build_custom_checkers(by_lang):
          'description': "The S in Terms of Service is capitalized"},
     ]) + comma_whitespace_rule
     html_rules = whitespace_rules + prose_style_rules + [
-        {'pattern': 'placeholder="[^{]',
+        {'pattern': 'placeholder="[^{](?:(?!\.com).)+$',
          'description': "`placeholder` value should be translatable.",
          'exclude_line': [('templates/zerver/register.html', 'placeholder="acme"'),
                           ('templates/zerver/register.html', 'placeholder="Acme or Aκμή"'),
-                          ('static/templates/settings/realm-domains-modal.handlebars',
-                           '<td><input type="text" class="new-realm-domain" placeholder="acme.com" /></td>'),
                           ("static/templates/user-groups-admin.handlebars",
                            '<input type="text" name="name" id="user_group_name" maxlength="100" placeholder="marketing" />')],
          'exclude': set(["static/templates/settings/emoji-settings-admin.handlebars",
                          "static/templates/settings/realm-filter-settings-admin.handlebars",
-                         "static/templates/settings/bot-settings.handlebars",
-                         "templates/zerver/email_log.html"]),
+                         "static/templates/settings/bot-settings.handlebars"]),
          'good_lines': ['<input class="stream-list-filter" type="text" placeholder="{{ _(\'Search streams\') }}" />'],
          'bad_lines': ['<input placeholder="foo">']},
-        {'pattern': "placeholder='[^{]",
```

Comment:
Why is it safe to remove this?  This is the other choice of quoting in comparison to the above...

Reply:
Oh, oops didn't see the `"` vs. `'`. Reverting.